### PR TITLE
Abort test execution unless elasticsearch is running.

### LINKED ISSUE #1303: Let default rake task fail if elasticsearch is not running
Elasticsearch running is a dependency of the default rake task. If it is not running, some tests are red. Let the task fail immediately if this dependency is not met.

### PR DESCRIPTION
**Pull request by eugenk**
This shall fix #1303.

### CODE REVIEW

File: lib/tasks/test.rake [hunk 1]
```diff
@@ -167,6 +167,17 @@ namespace :test do
     end
   end
 
+  desc 'abort execution if elasticsearch is not running'
+  task :abort_if_elasticsearch_is_not_running do
+    elasticsearch_port = ENV['ELASTIC_TEST_PORT'].present?
+    elasticsearch_port = '9250' unless elasticsearch_port.present?
+    unless port_open?('127.0.0.1', elasticsearch_port)
```
Comment (0robustus1) [inline lib/tasks/test.rake]: if `ENV['ELASTIC_TEST_PORT']` is actually `present?` the value of `elasticsearch_port` will be `true`, which is probably not what you want.

Comment (eugenk) [inline lib/tasks/test.rake]: Thanks, I was too quick there. Should be fixed by fa1ef27.

File: lib/tasks/test.rake [hunk 2]
```diff
@@ -167,6 +167,17 @@ namespace :test do
...
   end
 
+  desc 'abort execution if elasticsearch is not running'
+  task :abort_if_elasticsearch_is_not_running do
+    elasticsearch_port = ENV['ELASTIC_TEST_PORT']
+    elasticsearch_port = '9250' unless elasticsearch_port.present?
+    unless port_open?('127.0.0.1', elasticsearch_port)
+      message =
+        'Elasticsearch is not running. Please start it before running the tests'
+      raise message
```
Comment (0robustus1) [inline lib/tasks/test.rake]: Why do you raise an error instead of `puts`-ing to STDERR and `exit 1`?

Comment (eugenk) [inline lib/tasks/test.rake]: good point

**Comment by 0robustus1** [comment]
:+1:

**Pull request merged**